### ISSUE TITLE
T&C: data retention

### ISSUE THREAD

**Issue by BobHarper1** [OpenDataServices/cove#934]
https://github.com/OpenDataServices/cove/blame/master/cove/templates/terms.html#L97 says

> After a period of time we will delete those files from our server, meaning they are no-longer available for download.

Should this be more specific and say 7 days?

**Comment by Bjwebb** [comment]
Possibly.

I think those terms were deliberately written to be more general, so we could vary our practice (possibly between instances), without needing to write the T&Cs.

**Comment by robredpath** [comment]
Worth noting that we can override that more easily since we reworked the templating. If we want this changing for 360, we can do that. 

Can we commit to keeping files for 7 days in all cases - eg if we do a server rebuild? 

If we do change it, I'd suggest that we phrase it in a way that sets expectations but is realistic - perhaps "Files will be retained for a minimum of 7 days, before being scheduled for deletion. Deletion will occur within 30 days." - due to the ambiguity of what '7 days' means (168 hours? The 4am following the 7th midnight since the file was uploaded?), as I guess there is a risk that a deletion won't actually happen if there's a problem with the server and we don't want to commit too much!

**Comment by BobHarper1** [comment]
Ah, good point.

The wording on the front page (under 'More information') is:
> We delete all data older than 7 days from our servers daily, retaining none of the original data.

To me that would mean if data was uploaded on <today - 8 days>, it gets deleted. So I think the stricter wording in the T&C would cover other interpretations.

I don't think there's any need to change the wording on the front page, though (It also says _"We may also retain data in backups of our servers, which means on occasion, some data may be retained longer."_). But perhaps add a link to the Terms within that section so that users are fully informed?

I'd just thought that "a period of time" is too vague for T&Cs

**Comment by robredpath** [comment]
Yeah, I think it makes sense to keep the text under More Information concise and informal - but in the Ts&Cs we probably should specify a bit more what we're doing.

**Comment by BobHarper1** [comment]
Thanks will raise with 360

**Comment by Bjwebb** [comment]
> I'd just thought that "a period of time" is too vague for T&Cs

Still fairly vague, but the T&Cs do say: (emphasis mine)

> Information about how long files will be stored is made available to users directly through the interface, but will be for **no longer than one year** from the date of submission.

**Comment by BibianaC** [comment]
@robredpath is this still an issue?
